tests/small/test_frameworks_loader.py

```diff
@@ -1192,7 +1192,7 @@ class _CustomFramework(umake.frameworks.BaseFramework):
 
         def __init__(self):
             super().__init__(name="Custom", description="Custom uninstallable framework",
-                             category=umake.frameworks.MainCategory())
+                             category=umake.frameworks.MainCategory(), force_loading=False)
```

Comment:
I correct it! You are right it's better to have by default `force_loading` to `False`